src/_pytest/terminal.py

```diff
@@ -327,6 +327,9 @@ def showfspath(self, value):
     def showlongtestinfo(self):
         return self.verbosity > 0
 
+    def get_screen_width(self):
```

Comment:
I suggest using a property, for consistency with the properties above:

```python
@property
def screen_width(self):
    return self._screen_width
```

Reply:
Done.